### ISSUE TITLE
Unable to switch theme from dark to light in evaluation version [Accessibility concern]

### ISSUE THREAD

**Issue by deorth** [brisqapp/issue-tracker#93]
"I was hoping to evaluate the product, but I see that the ability to change theme from dark to light is considered a premium feature.

Since the product functionality is already limited without a license, locking the ability to change the theme seems excessive and makes me less interested in trying it, since I find it harder to read in dark mode. And while in my case the choice of theme is mostly preference (although my ADHD is probably a factor), for people with actual vision issues this would be a true limitation and a barrier to accessibility.

Please consider removing the lock on ability to manually change the theme from dark to light without a license. I think it help make your product more accessible."

**Comment by ashgrover** [comment]
@deorth 

Thanks for your feedback. I understand your concern. I'll definitely take your suggestion into consideration.
That said, the light theme is part of the paid plans and that was by design. I do have a 30 day money back guarantee if you'd like to try it out, and if you find that Brisqi doesn't fit your needs, just email me and I'll fully refund you.
